cdap-docs/_common/common_conf.py

```diff
@@ -106,6 +103,21 @@ def get_git_hash_timestamp():
         pass
     return git_hash, git_timestamp
 
+def get_git_build_vars():
```

Comment:
It is used to get the the Build Settings (GIT Branches) so that we can link to the correct Java source files.